series/brno-pyvo/events/2019-08-29.yaml

```diff
@@ -15,4 +15,8 @@ description: |
     we will not experience pub sauna as we did the last time.
 
 talks:
-    []
+  - title: Goodbye
+    description: |
+        ???
+    speakers: |
+        sha256:fd9f3f0f8e07d220a653add9f7dc9e052a3b232af0ca8c2eedb7aeeb5d62982b
```

Comment:
why?

Reply:
Thought it's a nice detail. Some personal "signature".
Any idea how to make it not rise the syntax error?

Reply:
Sure, this is YAML and this string contains `:`, so you need to [put in in quotes](http://blogs.perl.org/users/tinita/2018/03/strings-in-yaml---to-quote-or-not-to-quote.html).

Reply:
The actual issue is that `speakers` is a list. You need to put a `- ` in front.